frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -223,19 +236,28 @@ function PreviousValueComparison({
       : []),
     "",
   ];
+
   const detailCandidates = valueCandidates.map(valueStr => {
     if (isEmpty(valueStr)) {
       return comparisonDescStr;
     }
 
+    const descColor = isNightMode
```

Comment:
Same here, should the color be determined by `inTooltip` too?

https://github.com/metabase/metabase/blob/252c9423356cfb24315eb5c31a0e97da9a060787/frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.styled.tsx#L87-L89

Reply:
@WiNloSt I think this is same case as above, `PreviousValueComparison` seems to never be in the `Tooltip` so I suspect that the Emotion code here is mistaken/unused.

Reply:
so `descColor` is used inside `detailCandidates` which is used to declare `fullDetailDisplay`
https://github.com/metabase/metabase/blob/c681914357b325f169652c2500b287f5aa7fa690/frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx#L264

which is rendered inside the tooltip 
https://github.com/metabase/metabase/blob/c681914357b325f169652c2500b287f5aa7fa690/frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx#L312-L319

Doesn't it?

Reply:
Yep, I'm mistaken here - thanks for catching this!

Reply:
@WiNloSt committed the fix [here](https://github.com/metabase/metabase/pull/42347/commits/3d1f6687ef27a71249030aa598eeae3d96971205), it's a little bit more tricky than expected :)